Dockerfile.template

```diff
@@ -48,8 +48,18 @@ COPY root/usr/libexec/fix-permissions /usr/libexec/fix-permissions
 {{ spec.repo_enable_reason }}
 {%- endif %}
 RUN {{ spec.environment_setup }}
-{%- if not spec.stable and spec.staging_repo_setup %}
-{{ spec.staging_repo_setup }}
+{%- if not spec.stable and spec.has_devel_repo %}
+  {%- if config.os.id == 'centos' and spec.has_devel_repo.centos %}
+    yum-config-manager --add-repo https://cbs.centos.org/repos/sclo7-rh-postgresql{{ spec.version }}-rh-candidate/x86_64/os/ && \
+    echo gpgcheck=0 >> /etc/yum.repos.d/cbs.centos.org_repos_sclo7-rh-postgresql{{ spec.version }}-rh-candidate_x86_64_os_.repo && \
+  {%- elif config.os.id == 'rhel' %}
+    {%- if spec.has_devel_repo.rhel == 'brew' %}
+    yum-config-manager --add-repo http://download.devel.redhat.com/brewroot/repos/rhscl-{{ spec.rhscl_version.development }}-rh-postgresql{{ spec.version }}-rhel-7-build/latest/x86_64 && \
+    echo gpgcheck=0 >> /etc/yum.repos.d/download.devel.redhat.com_brewroot_repos_rhscl-{{ spec.rhscl_version.development }}-rh-postgresql{{ spec.version }}-rhel-7-build_latest_x86_64.repo && \
```

Comment:
Hm, I'm not sure about this.  I tried to fiddle enough ... but most probably this
won't work for dist-git release process, right?  Anything could be done better
@pkubat, without giving up RHEL CI entirely?

Reply:
I think its ok to have this in upstream as long as its only the unreleased images that are using it. And we change it to beta repositories as soon as they are available.